subcell_pipeline/analysis/compression_metrics/plot_compression_metrics.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+from typing import List, Union
+
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+
+from subcell_pipeline.analysis.compression_metrics.compression_analysis import (
+    COMPRESSIONMETRIC,
+)
+from subcell_pipeline.analysis.compression_metrics.constants import SIMULATOR_COLOR_MAP
+
+
+def plot_metrics_vs_time(
+    df: pd.DataFrame,
+    metrics: List[COMPRESSIONMETRIC],
+    figure_path: Union[Path, None] = None,
+    suffix: str = "",
+    compression_distance: float = 150.0,
+    use_real_time: bool = False,
+) -> None:
+    """
+    Plot metrics vs time.
+
+    Parameters
+    ----------
+    df
+        The input DataFrame.
+
+    metrics
+        The list of metrics to plot.
+
+    figure_path
+        The path to save the figure.
+
+    suffix
+        The suffix to append to the figure filename.
+        Defaults to "".
+
+    compression_distance
+        The compression distance in nm.
+        Defaults to 150.0.
+
+    use_real_time
+        Whether to use real time for the x-axis.
+        Defaults to False.
+    """
+    if figure_path is None:
```

Comment:
Expected behavior would be that is figure path is `None`, then the figures are not saved

Reply:
Fixed in [cd1ee6](https://github.com/simularium/subcell-pipeline/pull/58/commits/cd1ee6ac144ac0a9c368e04f2277e544a29e2aad)